service/v4/SugarWebServiceUtilv4.php

```diff
@@ -82,7 +82,8 @@ function get_module_view_defs($moduleName, $type, $view)
      * if the list should filter for favorites.  Should eventually update the SugarBean function as well.
      *
      */
-    function get_data_list($seed, $order_by = "", $where = "", $row_offset = 0, $limit=-1, $max=-1, $show_deleted = 0, $favorites = false)
+    // added a parameter to make it compatible with php7.X
+    function get_data_list($seed, $order_by = "", $where = "", $row_offset = 0, $limit=-1, $max=-1, $show_deleted = 0, $favorites = false, $single_select=false)
```

Comment:
Same again - Preferably using the same naming (or change it everywhere) $singleSelect=false